modules/Users/tpls/EditViewFooter.tpl

```diff
@@ -33,94 +34,103 @@
-                                <td scope="row" width="17%">
-                                    {$MOD.LBL_EMAIL_LINK_TYPE}:&nbsp;{sugar_help text=$MOD.LBL_EMAIL_LINK_TYPE_HELP WIDTH=450}
-                                </td>
-                                <td>
-                                    <select id="email_link_type" name="email_link_type" tabindex='410'>
-                                    {$EMAIL_LINK_TYPE}
-                                    </select>
-                                </td>
-                            </tr>
+<div id="email_options">
+    <table width="100%" border="0" cellspacing="1" cellpadding="0" class="edit view">
+        <tr>
+            <th align="left" scope="row" colspan="4">
+                <h4>{$MOD.LBL_MAIL_OPTIONS_TITLE}</h4>
+            </th>
+        </tr>
+        <tr>
+            <td scope="row" width="17%">
+                {$MOD.LBL_EMAIL}  {if $REQUIRED_EMAIL_ADDRESS}<span class="required"
+                                                                    id="mandatory_email">{$APP.LBL_REQUIRED_SYMBOL}</span> {/if}
+            </td>
+            <td width="83%">
+                {$NEW_EMAIL}
+            </td>
+        </tr>
+        <tr id="email_options_link_type" style='display:{$HIDE_FOR_GROUP_AND_PORTAL}'>
+            <td scope="row" width="17%">
+                {$MOD.LBL_EMAIL_LINK_TYPE}:&nbsp;{sugar_help text=$MOD.LBL_EMAIL_LINK_TYPE_HELP WIDTH=450}
+            </td>
+            <td>
+                <select id="email_link_type" name="email_link_type" tabindex='410'>
+                    {$EMAIL_LINK_TYPE}
+                </select>
+            </td>
+        </tr>
 
-                <tr>
-                    <td scope="row" width="17%">{$MOD.LBL_EDITOR_TYPE}</td>
-                    <td width="83%">
-                        <select id="editor_type" name="editor_type" tabindex='410'>
-                            {$EDITOR_TYPE}
-                        </select>
-                    </td>
-                </tr>
-                        </table>
-                        <button class="button" id="settingsButton" onclick="SUGAR.email2.settings.showSettings(); return false;"><img src="themes/default/images/icon_email_settings.gif" align="absmiddle" border="0"> {$APP.LBL_EMAIL_SETTINGS}</button>
-            </div>
+        <tr>
+            <td scope="row" width="17%">{$MOD.LBL_EDITOR_TYPE}</td>
+            <td width="83%">
+                <select id="editor_type" name="editor_type" tabindex='410'>
+                    {$EDITOR_TYPE}
+                </select>
+            </td>
+        </tr>
+    </table>
+    {if $ID}
+        <button class="button" id="settingsButton"
+                onclick="SUGAR.email2.settings.showSettings(getUserEditViewUserId()); return false;"><img
+                    src="themes/default/images/icon_email_settings.gif" align="absmiddle"
+                    border="0"> {$APP.LBL_EMAIL_SETTINGS}</button>
+    {/if}
+</div>
```

Comment:
No, it isn't accidental, added in initial commit. (8e51b0e9) In this commit fix only tab indents size.